src/test/java/org/example/jvspringbootfirstbook/controller/BookControllerTest.java

```diff
@@ -105,8 +105,8 @@ public void createBook_ValidCreateBookRequestDto_Success() throws Exception {
                 result.getResponse().getContentAsString(),
                 BookDtoWithoutCategoryIds.class
         );
-        Assertions.assertNotNull(actual);
-        Assertions.assertNotNull(actual.id());
+        Assertions.assertAll(() -> Assertions.assertNotNull(actual),
```

Comment:
You can group all assertions in assertAll() not only assertNotNull(), or only assertEquals() together.
  Assertions.assertAll(
                () -> Assertions.assertNotNull(actual),
                () -> Assertions.assertNotNull(actual.id()),
                ) -> Assertions.assertEquals(expected.title(), actual.title()))
        );